README.md

```diff
@@ -86,25 +89,25 @@ At the end of the day, the metric of success at Rapid River is: did you get the
 
 In general, the goal for full-time work at Rapid River is eight working hours per day, not counting breaks for meals, rest, etc. Some days it will be more than that; some days it will be less. If you find that you're consistently working a lot more than eight hours per day, you may be headed toward burn-out. Talk with your manager so that we can help you achieve a more balanced and sustainable schedule.
 
-Rapid Riverers work world-wide, which means that your hours may need to start earlier or (in more cases) go later than a traditional office job. Accommodate this in your daily schedule as much as possible by taking time for youself during slow periods to get up, stretch, go for a walk, get outside, etc. If you live with family or other loved ones, take time to be with them - they need you.
+Rapid Riverers work worldwide, which means that your hours may need to start earlier or (in more cases) go later than a traditional office job. Accommodate this in your daily schedule as much as possible by taking time for yourself during slow periods to get up, stretch, go for a walk, get outside, etc. If you live with family or other loved ones, take time to be with them * they need you.
```

Comment:
```suggestion
Rapid Riverers work worldwide, which means that your hours may need to start earlier or (in more cases) go later than a traditional office job. Accommodate this in your daily schedule as much as possible by taking time for yourself during slow periods to get up, stretch, go for a walk, get outside, etc. If you live with family or other loved ones, take time to be with them - they need you.
```
A little enthusiastic with the find-and-replace, I think. That last hyphen in the sentence was intended to be a hyphen.

Reply:
hahaha, you caught me there on find and replace... I surely missed that tiny bit